Formula/libzt.rb

```diff
@@ -0,0 +1,39 @@
+class Libzt < Formula
+  desc "ZeroTier: libzt -- An encrypted P2P networking library for applications"
+  homepage "https://www.zerotier.com"
+
+  url "https://download.zerotier.com/dist/sdk/libzt-1.3.0-source.tar.gz"
+  sha256 "0db5e1df43275d17162b986be29fa8c530695e5503f207d27739703730b5b23c"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "cmake", "--build", "."
+    system "make", "install"
+    cp "licenses/ZEROTIER-LICENSE.GPL-3", "#{prefix}/LICENSE"
+  end
+
+  def caveats
+    <<~EOS
+      Visit https://my.zerotier.com to create virtual networks and authorize devices.
+      Visit https://www.zerotier.com/manual.shtml to learn more about how ZeroTier works.
+      Visit https://github.com/zerotier/ZeroTierOne/tree/master/controller to learn how to run your own network controller (advanced).
+    EOS
+  end
+
+  test do
+    # Writes a simple test program to test.cpp which calls a library function. The expected output of this
+    # function is -2. This test verifies the following:
+    # - The library was installed correctly
+    # - The library was linked correctly
+    # - Library code executes successfully and sends the proper error code to the test program
+    (testpath/"test.cpp").write <<-EOS
+      #include<cstdlib>\n#include<ZeroTier.h>\nint main(){return zts_socket(0,0,0)!=-2;}
```

Comment:
These can all just use real linebreaks instead of \n

Reply:
Done